mod/item.php

```diff
@@ -277,10 +277,10 @@ function item_post(App $a) {
 		 */
 		/// @TODO use x($_REQUEST, 'foo') here
 		if (($api_source)
-			&& (! array_key_exists('contact_allow', $_REQUEST))
-			&& (! array_key_exists('group_allow', $_REQUEST))
-			&& (! array_key_exists('contact_deny', $_REQUEST))
-			&& (! array_key_exists('group_deny', $_REQUEST))) {
+			&& !array_key_exists('contact_allow', $_REQUEST)
+			&& !array_key_exists('group_allow', $_REQUEST)
+			&& !array_key_exists('contact_deny', $_REQUEST)
+			&& !array_key_exists('group_deny', $_REQUEST)) {
```

Comment:
All those can be advantageously replaced by `isset($_REQUEST['key'])` calls or even `x($_REQUEST, 'key') !== false`.